src/components/collections/collection-card.tsx

```
@@ -0,0 +1,144 @@
+          <div className="flex items-center gap-2 mt-3 text-xs text-muted-foreground">
+            <Folder className="h-3 w-3" />
+            <span>{collection.bookmarks_count} bookmarks</span>
+          </div>
+        </div>
+      </div>
+
+      {showMenu && (
+        <div
+          className="fixed inset-0 z-0"
+          onClick={() => setShowMenu(false)}
+        />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)